pandas/tests/internals/test_internals.py

```diff
@@ -1310,3 +1311,14 @@ def test_make_block_no_pandas_array():
     result = make_block(arr.to_numpy(), slice(len(arr)), dtype=arr.dtype)
     assert result.is_integer is True
     assert result.is_extension is False
+
+
+def test_add_column_with_pandas_array():
+    # GH 26390
+    df = pd.DataFrame({'a': [1, 2, 3, 4], 'b': ['a', 'b', 'c', 'd']})
```

Comment:
this test doesn't belong here, put instead in pandas/tests/frame/test_block_internals.py

Reply:
sure. doing that in the next commit